lib/shoulda/matchers/active_record/accept_nested_attributes_for_matcher.rb

```diff
@@ -174,6 +211,40 @@ def update_only_correct?
           verify_option_is_correct(:update_only, failure_message)
         end
 
+        def reject_if_correct?
+          failure_message = "reject_if #{should_or_should_not(@options[:reject_if])} be :#{@options[:reject_if]}, got "
+
+          if reject_if_option_is_a_callable?
+            message = "You need to use a symbol as reject_if param"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.